### ISSUE TITLE
'color' and 'rich' examples failing on master

### ISSUE THREAD

**Issue by flarnie** [facebook/draft-js#815]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
On current master in Chrome and Firefox, for me, I get the following error in both 'color' and 'rich' examples:
```
Uncaught TypeError: Cannot read property 'purgeUnmountedComponents' of undefined(…)
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. You can use this jsfiddle to get started: https://jsfiddle.net/stopachka/m6z0xn4r/.**

1) Clone master
2) `npm update && npm run build`
3) Open either `examples/rich/rich.html` or `examples/color/color.html`

**What is the expected behavior?**
They should render the examples.

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**

Draft un-released latest, in Chrome and Firefox and likely other browsers too.

**Comment by bowbowbow** [comment]
@flarnie 
I also experienced the same problem. 

But solve the problem after fix as bellow.

`before`
```javascript
// examples/rich/rich.html or examples/color/color.html
// line 29
<script src="../../node_modules/react/dist/react.min.js"></script>
```

`after`
```javascript
// examples/rich/rich.html or examples/color/color.html
// line 29
<script src="../../node_modules/react/dist/react.js"></script>
```

I think that the cause of the problem lies in the `gulpfile.js`.

**Comment by flarnie** [comment]
That's interesting - the minified version should still work. I'd like to figure out why this works before using that fix. Thanks for the clue @bowbowbow!

**Comment by spicyj** [comment]
I don't think this error should ever be possible with React.

@gaearon Have you seen this?

**Comment by spicyj** [comment]
Oh I see, it looks like this happens if you use minified react but unminified react-dom. Switching both to unminified also makes it work.

**Comment by spicyj** [comment]
This is related to React 15.4, not any Draft change.

**Comment by tobiasandersen** [comment]
I fixed this in #824 when re-adding the old (deprecated) examples, according to #816. Now all examples use the unminified version of both react and react-dom. Just let me know if that's not correct.
